styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -109,7 +109,7 @@ private BackfillsPayload getBackfills(RequestContext requestContext) {
 
     Stream<Backfill> backfills;
     try {
-      backfills = storage.backfills().stream();
+      backfills = storage.backfills().parallelStream();
```

Comment:
I mean that you could move `.parallelStream()` to just before line 130. I don't know, perhaps I'm being overly cautious about parallel streams.